docs/release_notes/v1.12.1.md

```diff
@@ -1,7 +1,34 @@
 # Dapr 1.12.1
 
-Update includes a fix for the Sentry authentication validator rejecting valid
-requests.
+
+This update contains the following security fixes:
+- Security: prevent Sentry and Injector from applying the `daprsystem` Configuration from a non control plane namespace.
```

Comment:
Please make these bullet points links to the headings below, so people can just "click"